iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/execution/executor/RegionExecutionResult.java

```diff
@@ -30,6 +30,17 @@ public class RegionExecutionResult {
   private TSStatus status;
   private boolean needRetry;
 
+  private RegionExecutionResult(boolean accepted, String message, TSStatus status) {
+    this.accepted = accepted;
+    this.message = message;
+    this.status = status;
+    this.needRetry = false;
+  }
+
+  public static RegionExecutionResult create(boolean accepted, String message, TSStatus status) {
+    return new RegionExecutionResult(accepted, message, status);
+  }
```

Comment:
What's the difference between RegionExecutionResult.create and calling new RegionExecutionResult directly?

Reply:
`RegionExecutionResult.create` provides a layer of abstraction for object creation, encapsulating the details of object construction. Directly calling `new RegionExecutionResult` is a straightforward way to create objects, suitable for simpler scenarios. It makes the code more intuitive but offers less flexibility.